docs/constants/navigation.js

```diff
@@ -141,10 +141,11 @@ const general = [
       'Web',
       [
         makePage('workflow/web.mdx'),
-        makePage('guides/progressive-web-apps.mdx'),
         makePage('distribution/publishing-websites.mdx'),
-        makePage('distribution/publishing-websites-webpack.mdx'),
-        makePage('guides/customizing-webpack.mdx'),
+        makePage('guides/customizing-metro.mdx'),
+        makePage('guides/progressive-web-apps.mdx'),
+        // makePage('distribution/publishing-websites-webpack.mdx'),
+        // makePage('guides/customizing-webpack.mdx'),
```

Comment:
I understand we are hiding them from the main navigation structure, but since we are keeping them around, I think a good practice would be to move them under Archive: https://docs.expo.dev/archive/ (so that we can keep them around for legacy reasons)?

Reply:
Nice, yeah I just wasn't sure where to put them